QuantConnect.BybitBrokerage.Tests/BybitBrokerageHistoryProviderTests.cs

```diff
@@ -107,7 +107,8 @@ private static TestCaseData[] InvalidHistory
         [Test]
         [TestCaseSource(nameof(NoHistory))]
         [TestCaseSource(nameof(InvalidHistory))]
-        public virtual void GetEmptyHistory(Symbol symbol, Resolution resolution, TimeSpan period, TickType tickType)
+        public virtual void GetEmptyHistory(Symbol symbol, Resolution resolution, TimeSpan period, TickType tickType,
+            bool throwsException)
```

Comment:
In every test case source, the value of `throwsException` is false, so is it necessary?

Reply:
We don't :)